subprojects/internal-performance-testing/internal-performance-testing.gradle.kts

```diff
@@ -67,6 +67,9 @@ dependencies {
     implementation(testLibrary("jetty"))
     implementation(testFixtures(project(":core")))
     implementation(testFixtures(project(":toolingApi")))
+    implementation("org.gradle.profiler:gradle-profiler:1.0-SNAPSHOT") {
+        exclude(group = "com.google.guava", module = "guava")
```

Comment:
how much effort would it be to unify the two versions?

Reply:
We need to document the reasons here.

Reply:
We don't need to exclude it any more, since we now use the Android version in Gradle profiler as well.

Reply:
I removed the exclusion.